projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -138,18 +138,21 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
    * @return: true or false
    */
   canCancelRequest(): boolean {
-    // TODO: manage more complex cases according to specifications (when implemented in backend or invenio-circulation)
-    return this.transaction.metadata.state === LoanState.PENDING;
+    // No permission API in backend
+    // Allowed when loan state is PENDING or ITEM_IN_TRANSIT_FOR_PICKUP according to actions.md
+    return (this.transaction.metadata.state === LoanState.PENDING
+      || this.transaction.metadata.state === LoanState.ITEM_IN_TRANSIT_FOR_PICKUP) ? true : false;
```

Comment:
Why use ternary instruction ? Isn't it enough `return this.transaction.metadata.state === LoanState.PENDING || this.transaction.metadata.state === LoanState.ITEM_IN_TRANSIT_FOR_PICKUP` ?